docs/src/examples.md

```diff
@@ -148,3 +148,45 @@ for stage in 1:PSRI.total_stages(data)
     println("Thermal generator 2 generation capacity at stage $stage $(therm_gen.generation_capacities[2])")
 end
 ```
+
+## Determining subsystem from a certain gauging station
+
+In this example we will demonstrate how to chain different relationship maps. That will be achieved by determining a subsystem from a certain gauging station through HydroPlants parameters. The program will initiate by the standard reading procedure:
+```@example sys_by_gaug
+import PSRClassesInterface
+const PSRI = PSRClassesInterface
+
+PATH_CASE_EXAMPLE_GAUGING = joinpath(pathof(PSRI) |> dirname |> dirname, "test", "data", "caso2")
+
+data = PSRI.initialize_study(
+    PSRI.OpenInterface(),
+    data_path = PATH_CASE_EXAMPLE_GAUGING
+)
+```
+
+Next, the maps between hydroplants and systems, and hydroplants and gauging stations are retrieved by the `get_map` method:
+```@example sys_by_gaug
+hyd2sys = PSRI.get_map(data, "PSRHydroPlant","PSRSystem")
+hyd2gau = PSRI.get_map(data, "PSRHydroPlant","PSRGaugingStation")
```

Comment:
You can do 
```
gau2hyd = PSRI.get_map(data, "PSRHydroPlant","PSRGaugingStation")
```
and then `gau2sys[I]` becomes `hyd2sys[gau2hyd[I]]`.

In the end you can simply use `gau2sys` to query all systems from gauging stations

Reply:
The is no relation `gau2hyd` (`.get_map(data, "PSRGaugingStation", "PSRHydroPlant")`), only `hyd2gau` (gau->hyd isn't defined in _RELATIONS). If I understood the method correctly, `I` would have to be the index of the hydroplant that relates to the gauging station we are interested in, which would require an extra step to be found

Reply:
Indeed! In this case I think that the example I was thinking of is not possible. One GuagingStation can relate to multiple systems. You can write the example only relating HydroPlants to Systems.